src/api/external/projectApi.ts

```diff
@@ -0,0 +1,23 @@
+import Axios from 'axios';
+import { Dispatch } from 'redux';
+
+import { addNotification } from '../../redux/actions/notificationActions';
+import i18n from '../../translations/i18n';
+import { API_URL, ApiServiceUrl, StatusCode } from '../../utils/constants';
+
+const PROJECT_SERVICE_BASE_URL = `${API_URL}${ApiServiceUrl.PROJECT}`;
+export const PROJECT_SEARCH_URL = `${PROJECT_SERVICE_BASE_URL}/insert-resource`;
+
+export const searchProjects = async (query: string, dispatch: Dispatch) => {
+  try {
+    const response = await Axios.get(`${PROJECT_SEARCH_URL}?${query}`);
```

Comment:
jeg kan være enig i at det er lurt å lagre path names som konstanter. ville foretrukket noe sånt isåfall:
`Axios.get(`/${ApiServiceUrl.PROJECT}?{query}`) (nå som vi har API_URL som base i Axios)

Reply:
yup